src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -1503,6 +1514,38 @@ public FormValidation doCheckMinimumNumberOfInstances(@QueryParameter String val
             return FormValidation.error("Minimum number of instances must be a non-negative integer (or null)");
         }
 
+        public FormValidation doCheckMinimumNoInstancesActiveTimeRangeFrom(@QueryParameter String value) {
+            try {
+                MinimumNumberOfInstancesTimeRangeConfig.validateLocalTimeString(value);
+                return FormValidation.ok();
+            } catch (IllegalArgumentException e) {
+                return FormValidation.error("Please enter value in format 'h:mm a' or 'HH:mm'");
+            }
+        }
+
+        public FormValidation doCheckMinimumNoInstancesActiveTimeRangeTo(@QueryParameter String value) {
+            try {
+                MinimumNumberOfInstancesTimeRangeConfig.validateLocalTimeString(value);
+                return FormValidation.ok();
+            } catch (IllegalArgumentException e) {
+                return FormValidation.error("Please enter value in format 'h:mm a' or 'HH:mm'");
+            }
+        }
+
+        // For some reason, all days will validate against this method so no need to repeat for each day.
+        public FormValidation doCheckMonday(@QueryParameter boolean monday,
```

Comment:
Maybe a different name if it's valid for all days?

Reply:
Ideally, yes, but it's some sort of Jelly magic based on the field names.